apps/chat/channels.py

```diff
@@ -361,9 +363,14 @@ def _reply_voice_message(self, text: str):
         text, extracted_urls = strip_urls_and_emojis(text)
 
         voice_provider = self.experiment.voice_provider
+        synthetic_voice = self.experiment.synthetic_voice
+        if self.experiment.use_processor_bot_voice and self.bot.processor_experiment.voice_provider:
+            voice_provider = self.bot.processor_experiment.voice_provider
```

Comment:
could `bot` be None here?

Reply:
Bot cannot, but `bot.processor_experiment` can actually be since we call `send_message_to_user` from another place in the code where the bot hasn't generated a response, but rather we give it what it should say, so the TopicBot's `processor_experiment` will be `None`. I'll update the tests to cover this. Actually, there is a test, but the tests are not running against a multi bot setup, only a single bot setup, which is why everything is hunky dory.